docs/src/design/decoder/main.md

```diff
@@ -287,9 +287,9 @@ In the above diagram, `blk` is the ID of the *split* block which is about to be
 When the VM executes a `SPLIT` operation, it does the following:
 
 1. Adds a tuple `(blk, prnt, 0)` to the block stack table.
-2. Pops the stack and:
-   a. If the popped value is $1$, adds a tuple `(blk, true_branch_hash, 0, 0)` to the block hash table.
-   b. If the popped value is $0$, adds a tuple `(blk, false_branch_hash, 0, 0)` to the block hash table.
+2. Pops the stack and:\
+   a. If the popped value is $1$, adds a tuple `(blk, true_branch_hash, 0, 0)` to the block hash table.\
+   b. If the popped value is $0$, adds a tuple `(blk, false_branch_hash, 0, 0)` to the block hash table.\
```

Comment:
Why do we need this extra `\` here (and also in the similar paragraph on line 306?

Reply:
Without it this nested list is looking like this:
```
2. Pops the stack and: a. If the popped value is 1, adds a tuple (blk, true_branch_hash, 0, 0) to the block hash table. b. If 
   the popped value is 0, adds a tuple (blk, false_branch_hash, 0, 0) to the block hash table. c. If the popped value is 
   neither 1 nor 0, the execution fails.
```
Extra `\` here ads additional line break and created nested list:
```
2. Pops the stack and:
   a. If the popped value is 11, adds a tuple (blk, true_branch_hash, 0, 0) to the block hash table.
   b. If the popped value is 00, adds a tuple (blk, false_branch_hash, 0, 0) to the block hash table.
   c. If the popped value is neither 11 nor 00, the execution fails.
``` 
At least this is how it is displayed for me, I still need to check whether it is displayed correctly for others.

Reply:
Ah - makes sense!